invenio_records_rest/links.py

```diff
@@ -2,6 +2,9 @@
 #
 # This file is part of Invenio.
 # Copyright (C) 2015, 2016 CERN.
+# Copyright (C) 2017 Swiss Data Science Center (SDSC)
+# A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
+# Eidgenössische Technische Hochschule Zürich (ETHZ).
```

Comment:
I don't think these two extra lines should be added. The copyright should just state who the copyright holder is (must be a legal entity) - e.g. either it's SDSC (if SDSC is a legal entity) or it is EPFL + ETHZ (if SDSC is not a legal entity)?
For crediting we can add in AUTHORS.rst that SDSC is a partnership between EPFL/ETHZ.

Reply:
@lnielsen this is the official way for now according to EPFL/ETH layers for now.

Reply:
The problem is that both institutions need to be mentioned in full name. It should be easier with the NOTICE file when the MIT license is applied.